spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderTests.java

```diff
@@ -107,6 +116,19 @@ public void init() {
 		}
 	}
 
+	@After
+	public void cleanup() {
+		try {
+			Seq<String> topics = ZkUtils.getAllTopics(kafkaTestSupport.getZkClient());
+			for (String topic: scala.collection.JavaConversions.seqAsJavaList(topics)) {
+				AdminUtils.deleteTopic(kafkaTestSupport.getZkClient(), topic);
+			}
+		}
+		catch (Exception e) {
+			log.error(e.getMessage());
```

Comment:
I believe the scope of this PR is intended to delete the topics on **embedded** kafka server in `KafkaBinderTests` in which the test methods share the **same** embedded kafka server. 
That's the reason I went ahead with delete all topics during the cleanup.
Also, I think the KafkaBinderTests always runs with the embbedded kafka server based on what we set as `ClassRule` there. What am I missing?